crates/utils/src/task_manager/tasks.rs

```diff
@@ -54,10 +50,7 @@ enum_with_iter! {
 }
 
 /// All edges of task graph, the first item in each pair must be shut down before the second item
-pub const ALL_EDGES: [(TaskName, TaskName); 9] = [
-    (TaskName::ConflictCheckedMpmc, TaskName::CmdWorker),
-    (TaskName::CmdWorker, TaskName::CompactBg),
-    (TaskName::CmdWorker, TaskName::KvUpdates),
+pub const ALL_EDGES: [(TaskName, TaskName); 6] = [
```

Comment:
How should we track this task if we remove the `CompactBg` from the `ALL_EDGES`? Is it possible that this task will fall off the task manager's radar?

Reply:
It could be canceled safely when Xline exits, because it doesn't have an dependent tasks.